Lab5.py

```diff
@@ -0,0 +1,87 @@
+class Flower:
+    def __init__(self, name, height, size, color, price, quantity, delivery_rate):
+        self.name = name
+        self.height = height
+        self.size = size
+        self.color = color
+        self.price = price
+        self.quantity = quantity
+        self.delivery_rate = delivery_rate
+
+
+class FlowerShop:
+    def __init__(self):
+        self.stock = []
+
+    def add_flower(self, flower, quantity):
+        self.stock.extend([flower] * quantity)
+
+    def remove_flower(self, flower):
```

Comment:
recommendation
change 'remove_flower' to 'remove_flowers'

Reply:
why remove_flowers? it emoves only one flower